HW_8/merge.py

```diff
@@ -0,0 +1,40 @@
+
+import student_info as si
+import cabinet_info as cab
+
+
+def option():
+    print("\nВас приветствует программа мониторинга процессов обучения студентов!")
+    ch = int(input('Введите что хотите сделать: \n \
+    1: Поиск ID студента по фамилии \n \
+    2: Посмотреть класс  и место которое занимает  студент \n \
+    3: Выход\n \
+    Ваш выбор: '))
+
+    if ch == 1:
+        Surn = str(input("Введите фамилию ученика: "))
+        if Surn in si.stud_card['Фамилия']:
+            index = si.stud_card['Фамилия'].index(Surn)
+        print(f"{si.stud_card['ID'][index]}, {si.stud_card['Имя'][index]},{si.stud_card['Фамилия'][index]}\n,{si.stud_card['Дата рождения'][index]}, {si.stud_card['Успеваемость'][index]}")
+        print('\nХотите выполнить какую-то другую операцию??? Y или N: ')
+        num = input()
+        if num == 'Y' or 'y' or 'У' or 'у':
+            option()
+        exit()
```

Comment:
Не стоит использовать эту функцию для выхода из приложения. Да и вообще так прерывать работу программы не рекомендуется, но если очень хочется, то лучше так:
```suggestion
        sys.exit()
```
С импортом соответствующей библиотеки